crates/solvers/src/api/routes/solve/dto/solution.rs

```diff
@@ -67,16 +67,6 @@ impl Solutions {
                                         buy_token_balance: BuyTokenBalance::Erc20,
                                         signing_scheme,
                                         signature,
-                                        pre_interactions: trade
```

Comment:
> Remove pre-interactions for jit orders (for some reason we didn't populate post interactions...

I don't remember if it was a conscious decision to not populate post interactions for JIT orders but they are not strictly necessary, right?. The solver is the one that builds and uses a JIT order. With that information they should be able to execute anything that would make sense as a post-interaction as part of the regular interactions in the middle.
This is not the case for pre-interactions, though.

Reply:
Sure, but I don't think this nuanced reasoning justifies this inconsistency (technically a self-hosted driver could also execute whatever call they want before settle).

Practically, no one is using this order type (and for sure not with pre-interactions), so I still think it's in the spirit of #simplification to remove it.

Reply:
> technically a self-hosted driver could also execute whatever call they want before settle

How would they ensure that their interaction happens in the same block as their settlement?
Wouldn't they need tight block builder integration for that?

I'm fine with removing it. Just wanted to point possible reasons for why it is like it is.